xml/docu_styleguide.structure.xml

```diff
@@ -647,6 +647,24 @@ data  home  lost+found  opt     run   srv      tmp&lt;/screen&gt;</screen>
       </listitem>
      </itemizedlist>
     </listitem>
+    <listitem>
+      <para>To enable automatic syntax highlighting for programming languages or formats, add a language 
```

Comment:
```suggestion
      <para>To enable automatic syntax highlighting for programming languages or
      formats, add a <tag class="attribute">language</tag> 
```

Reply:
@dariavladykina do you add the `<tag>` element too? That's actually also important.

Reply:
Thanks for your review @tomschr ! Could you please help me understand where to add the `<tag>` element?

Reply:
Yes, see above. It's the "_...for programming languages or formats, add a **language** attribute_". The bold part needs the respective tag (also see above).